igvm/commands.py

```diff
@@ -224,32 +228,48 @@ def change_address(vm_hostname, new_address, offline=False):
                     vm.dataset_obj['igvm_operation_mode'])
             )
 
-        old_address = vm.dataset_obj['intern_ip']
         new_address = ip_address(new_address)
 
-        if old_address == new_address:
+        if vm.dataset_obj['intern_ip'] == new_address:
```

Comment:
What about IPv6 address. Won't it be broken after changing the network ?

Reply:
It will be. We could move functionality of `add_ipv6_address` script to Serveradmin_extras to solve this once and for all.

Reply:
I consider this one resolved because it is done as good as possible now, and once Serveradmin auto-assigns IPv6 this issue will be fully resolved.